types/luxon/src/duration.d.ts

```diff
@@ -78,6 +88,19 @@ export type DurationLike = Duration | DurationLikeObject | number;
 
 export type DurationMaybeValid = CanBeInvalid extends true ? (Duration<Valid> | Duration<Invalid>) : Duration;
 
+export interface ToFormatOptions {
```

Comment:
This name seems too ambiguous. `DateTime`, `Duration`, and `Interval` have a `toFormat` method with options.

Maybe `DurationFormatOptions`?